patches/master_patch.patch

```diff
@@ -433,6 +433,19 @@ index a5ea92f2878459b49f88d4895c7d18f2e157dd6c..9cffca0352713bce12b4035c02612cd7
      }
    }
  
+diff --git a/chrome/browser/renderer_preferences_util.cc b/chrome/browser/renderer_preferences_util.cc
+index ef8350dd1f99470c0e535f39ac48b8ff867d1105..200803d01a0ee6bce2d5b3b7d97ac735a96f6ef1 100644
+--- a/chrome/browser/renderer_preferences_util.cc
++++ b/chrome/browser/renderer_preferences_util.cc
+@@ -136,7 +136,7 @@ void UpdateFromSystemSettings(content::RendererPreferences* prefs,
+     prefs->caret_blink_interval = interval;
+ #endif
+ 
+-#if defined(USE_AURA) && defined(OS_LINUX) && !defined(OS_CHROMEOS)
++#if defined(USE_AURA) && defined(MUON_CHROMIUM_BUILD) && defined(OS_LINUX) && !defined(OS_CHROMEOS)
+   views::LinuxUI* linux_ui = views::LinuxUI::instance();
+   if (linux_ui) {
+     if (ThemeServiceFactory::GetForProfile(profile)->UsingSystemTheme()) {
```

Comment:
we are using `BuildGtkUi` for `LinuxUI` so just comment out this check and put a comment says why we are doing this

Reply:
comment out this whole block will lose the attributes passing from GTK UI to Webkit